pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +265,37 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) parallelPullSupportExists(serializeImagePulls util.BoolFlag) bool {
+	useParrallelPull, err := dm.isDockerAPIVersionCompatible(minimumDockerAPIVersionForParallelPulls)
```

Comment:
This variable name isn't correct. It sounds like if this is true, we should use parallel pulls. I would call it `canParallelPull` or `parallelSupportExists` or something like that.